src/views/DandisetLandingView/DandisetPublish.vue

```diff
@@ -305,6 +305,7 @@ import moment from 'moment';
 
 import { dandiRest, loggedIn as loggedInFunc, user as userFunc } from '@/rest';
 import store from '@/store';
+import router from '@/router';
```

Comment:
I didn't know you could just import the router like this and use it, that seems cleaner then using the `ctx.root.$router` variable https://github.com/dandi/dandiarchive/blob/master/src/views/MetadataView/Meditor.vue#L310.